src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/OffenderSentence.kt

```diff
@@ -185,6 +189,7 @@ class OffenderSentenceBuilder(
     sled2Calc = sled2Calc,
     startDate2Calc = startDate2Calc,
   )
+    .let { repository.save(it) }
```

Comment:
save this early so the id is generated - makes using DSL simpler - you can access the ID immediately rather than waiting for the building to return